### PR TITLE
chore: README badges 

### PR DESCRIPTION
**Pull request by alexanderbez**
Closes: #1423

[RENDERED](https://github.com/osmosis-labs/osmosis/blob/bez/1423-readme-badges/README.md)

### CODE REVIEW

**Comment by alexanderbez** [comment]
@ValarDragon or @daniel-farina, is there an SVG or PNG that I can use in the header to display the logo?

File: README.md [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
-
-Whereas most Cosmos zones have focused the  ir incentive scheme on the delegators, Osmosis attempts to align the interests of multiple stakeholders of the ecosystem such as LPs, DAO members, as well as delegators. One mechanism that is introduced is how staked liquidity providers have sovereign ownership over their pools, and through the pool governance process allow them to adjust the parameters depending on the pool’s competition and market conditions. Osmosis is a sovereign Cosmos zone that derives its sovereignty not only from its application-specific blockchain architecture but also the collective sovereignty of the LPs that has aligned interest to different tokens that they are providing liquidity for.
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://img.shields.io/badge/repo%20status-WIP-yellow.svg?style=flat-square)](https://www.repostatus.org/#wip)
```
Comment (ValarDragon) [inline README.md:3]: Should this be Active?

**Review by ValarDragon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (alexanderbez) [reply README.md:3]: If you think so :)

Comment (alexanderbez) [reply README.md:3]: We're on mainnet, so I think so.

**Comment by jonator** [comment]
@alexanderbez Is this what you're looking for?

**Pull request merged**